import-service/lib/services/import/import-service.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-restricted-syntax */
+import AWS from 'aws-sdk';
+import csvParser from 'csv-parser';
+
+const BUCKET = 'arsiompeshkonodejsinaws2-product-import';
+
+export function getSignedUrl(fileName: string): string {
+  const s3 = new AWS.S3({ region: 'eu-west-1' });
+
+  const params = {
+    Bucket: BUCKET,
+    Key: `uploaded/${fileName}`,
+    Expires: 60,
+    ContentType: 'text/csv',
+  };
+  const url = s3.getSignedUrl('putObject', params);
+
+  return url;
+}
+
+async function parseRecord(record: any): Promise<void> {
+  const s3 = new AWS.S3({ region: 'eu-west-1' });
+
+  return new Promise((resolve, reject) => {
+    const recordKey = record.s3.object.key;
+    const parsedRecordKey = recordKey.replace('uploaded', 'parsed');
```

Comment:
It's better to declare the variable inside the 'end' callback as you can catch the error event and this variable will stay unnecessary.